addons/spreadsheet_account/models/account.py

```diff
@@ -118,6 +118,9 @@ def spreadsheet_fetch_debit_credit(self, args_list):
                 continue
             MoveLines = self.env["account.move.line"].with_company(company_id)
             query = MoveLines._search(domain)
+            if query == []:
+                results.append({"credit": 0, "debit": 0})
+                continue
```

Comment:
I think I fixed the same issue 2 weeks ago with https://github.com/odoo/odoo/pull/163165/files
It's the same issue/fix, right ?

Reply:
seems like it - I forgot about the task as it was a subtask without any project set ...